### PR TITLE
Full signature for namedtuple

### PR DESCRIPTION
**Pull request by elazarg**
Issue #533

### CODE REVIEW

File: stdlib/3/typing.pyi [hunk 1]
```diff
@@ -418,3 +417,11 @@ class Pattern(Generic[AnyStr]):
 # Functions
 
 def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+
+
+# Type constructors
+
+# NamedTuple is special-cased in the type checker; the initializer is ignored.
+def NamedTuple(typename: str, fields: Iterable[Tuple[str, Any]], *,
+               verbose: bool = ..., rename: bool = ..., module: str = None) -> Type[tuple]: ...
```
Comment (elazarg) [inline stdlib/3/typing.pyi]: `fields` can be more exact as `Iterable[Tuple[str, Union[str, Type]]]`  or something like that.

**Comment by elazarg** [comment]
(`NamedTuple` definition moved down to allow using `Iterable`)

**Comment by gvanrossum** [comment]
Can you also do these for 2.7?

Also, please be careful about not adding extra blank lines (read your own review to find where they are).

File: stdlib/3/typing.pyi (hunk 1 above)
Comment (gvanrossum) [inline stdlib/3/typing.pyi]: Hm, a type annotation doesn't have to be a type (see https://github.com/python/typing/issues/136). So better leave this as is, or use Union[str, Any].

File: stdlib/3/typing.pyi [hunk 2]
```diff
@@ -418,3 +417,10 @@ class Pattern(Generic[AnyStr]):
 # Functions
 
 def get_type_hints(obj: Callable) -> dict[str, Any]: ...
+
+# Type constructors
+
+# NamedTuple is special-cased in the type checker; the initializer is ignored.
+def NamedTuple(typename: str, fields: Iterable[Tuple[str, Any]], *,
+               verbose: bool = ..., rename: bool = ..., module: str = None) -> Type[tuple]: ...
+
```
Comment (gvanrossum) [inline stdlib/3/typing.pyi]: Kill this blank line please.

**Comment by gvanrossum** [comment]
There's also 2.7/typing.pyi.

**Comment by gvanrossum** [comment]
Fingers crossed!

**Pull request merged**